assets/javascripts/discourse/components/ai-llms-list-editor.gjs

```diff
@@ -112,17 +112,9 @@ export default class AiLlmsListEditor extends Component {
   }
 
   localizeUsage(usage) {
-    if (usage.type === "ai_persona") {
-      return i18n("discourse_ai.llms.usage.ai_persona", {
-        persona: usage.name,
-      });
-    } else if (usage.type === "automation") {
-      return i18n("discourse_ai.llms.usage.automation", {
-        name: usage.name,
-      });
-    } else {
-      return i18n("discourse_ai.llms.usage." + usage.type);
-    }
```

Comment:
I don't think we needed all this complexity here, I get that we think name is more accurate than persona for automation, but this forces us to always have a special case for it (and created this bug, as we forgot the special case at another place). Maybe we could also think of a more generic name, like: identifier.